hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleAccountState.java

```diff
@@ -75,6 +83,9 @@
 	private ByteString alias = DEFAULT_ALIAS;
 	private int autoAssociationMetadata;
 	private int numContractKvPairs;
+	private Map<EntityNum, Long> cryptoAllowances = new TreeMap<>();
```

Comment:
We can initialize to `Collections.emptyMap()` to avoid creating unused maps.

Reply:
Used as suggested and created new `TreeMap` while inserting elements into the maps during `deserialize` . Will be doing the same during `ApproveAllowance` and `AdjustAllowance` transactions to insert or modify elements in the maps.